bot/exts/evergreen/fun.py

```diff
@@ -60,15 +60,13 @@ def __init__(self, bot: Bot) -> None:
     @commands.command()
     async def roll(self, ctx: Context, num_rolls: int = 1) -> None:
         """Outputs a number of random dice emotes (up to 6)."""
-        output = ""
         if num_rolls > 6:
             num_rolls = 6
-        elif num_rolls < 1:
-            output = ":no_entry: You must roll at least once."
-        for _ in range(num_rolls):
-            dice = f"dice_{random.randint(1, 6)}"
-            output += getattr(Emojis, dice, '')
-        await ctx.send(output)
+
+        dice = (f"dice_{random.randint(1, 6)}" for _ in range(num_rolls))
+        output = " ".join(getattr(Emojis, die, '') for die in dice)
+
+        await ctx.send(output or ":no_entry: You must roll at least once.")
```

Comment:
I don't like this. You've turned 6 lines into 3, and introduced more complexity. I don't mind your code, but I prefered when it was 6 lines - it had better readability.

Our code style is to not have a newline after docstrings for functions and coroutines, by the way. I'm not sure why the linter let this through, I guess it's not set up with the same flake8-docstrings rules as we have on `bot`.

Reply:
While I'm not fond of the general logic in the code — looping and creating new string objects every time — I still get your point about readability. I'll revert to the original solution, with the slight alterations needed to add the spaces.

Also, there is no newline after the docstring. I still had this part on top: 
```py
if num_rolls > 6:
             num_rolls = 6
```

Reply:
oh okay, just misread that then. my bad.

Reply:
I see what you're saying. Either way, I pushed a new commit that returned to the original logic and simply added the spaces. Sadly didn't see that you added that "for the record" part before finishing it.